src/resdk/tables/variant.py

```diff
@@ -279,20 +280,14 @@ def construct_index(self, row) -> str:
 
         # REF & ALT
         ref = row["REF"]
-        alt = row.get("ALT", "?")
-        if alt in ["A", "C", "G", "T"]:
+        alt = str(row.get("ALT", "?"))
+        if re.match(r"^[ACGT]+$", alt):
             index += f"_{ref}>{alt}"
 
-        # Amino acid change
-        aa_change = row.get("HGVS.p", "?")
-        aa_change = self.strip_amino_acid(aa_change)
-        if aa_change:
-            index = f"{index}_{aa_change}"
-
         return index
 
     @staticmethod
-    def encode_mutation(row) -> int:
+    def _encode_mutation(row) -> int:
```

Comment:
After the latest change to process `mutations-table`, variants are reported only if they have been detected. This means that you cannot get wild type variant in input samples - you will never have a zero in `tables.variants`. I suggest that we remove encoding for wild type.